services/property-svc/internal/property-value/aggregate/aggregate.go

```diff
@@ -72,7 +75,12 @@ func (a *PropertyValueAggregate) onPropertyValueUpdated(evt hwes.Event) error {
 		return err
 	}
 
-	a.PropertyValue.Value = payload.Value
+	if payload.Change.ValueRemoved {
+		a.PropertyValue.Value = nil
+		return nil
+	}
+
+	payload.Change.Apply(a.PropertyValue.Value)
```

Comment:
We do not need to re-assign this due to the reference above?

Reply:
which reference above?

Reply:
Line #67

Reply:
that's in onPropertyValueCreated, not onPropertyValueUpdated